x/chainmanager/types/params.go

```diff
@@ -93,35 +93,67 @@ func (p Params) String() string {
 
 // Validate checks that the parameters have valid values.
 func (p Params) Validate() error {
-	if err := validateAccAddress(MaticTokenAddress, p.ChainParams.MaticTokenAddress); err != nil {
+	addr, err := sdk.AccAddressFromHex(p.ChainParams.MaticTokenAddress)
```

Comment:
Not sure what should go here in validate () method. @jdkanani / @anilCSE any suggestions?

Reply:
we are validating the chain params. what more should we add? @murthyvitwit

Reply:
We need to decide validation cases. now we are just converting String to AccAddress.

Reply:
Anyway, I will add an issue for this.